docs/learn/learn-staking-advanced.md

```diff
@@ -0,0 +1,385 @@
+---
+id: learn-staking-advanced
+title: Advanced staking
+sidebar_label: Advanced staking
+description: Advanced staking information
+keywords: [staking, stake, nominate, nominating, NPoS, proxies]
+slug: ../learn-staking-advanced
+---
+
+import RPC from "./../../components/RPC-Connection"
+
+This page is meant to be an advanced guide to staking with {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Staking Proxies
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is built using
+[substrate](https://substrate.io/), a modular system to efficiently build blockchains. Within each module or **pallet**, one can **call** different
+functions that have similar logic. For example, the staking pallet contains all functionalities
+related to staking such as bonding or unbonding funds. The combined information of pallets and calls
+constitutes an **extrinsic**, i.e. a transaction that is executed from outside the chain but that
+triggers an event on the chain. Continuing with the staking example, within the staking pallet
+a nominator can bond funds and nominate some validators. The signature of such
+extrinsic might lead to an event on the chain such as a reward payout to that nominator at the end of an era; this is an event inside the chain. This way of having
+transactions categorized within pallets and functionalities makes it possible to create accounts
+having special permissions also called **proxy accounts**.
+
+![staking](../assets/staking/stash-stakingProxy.png)
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
```

Comment:
With the above comment in mind, this feels like a natural starting point for the section.

Reply:
Made links to dedicated section at the start of the page